Open-Source-Pundits/nutshell
# Create app html

# Description

Created landing page, which links to all the individual pages with their containers. Did some basic styling.

Fixes #25 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

- git fetch --all
- check out into the create-app-html branch
- npm start
- make sure the links on the home page are clickable (side nav buttons do not function yet)

# Checklist:
